app/models/search_facet.rb

```diff
@@ -283,6 +283,11 @@ def is_numeric?
     self.data_type == 'number'
   end
 
+  # for now, assume it's time if it's numeric and has a base of years
```

Comment:
Is `self.unit` necessarily a year here?  If not, this might be clearer:

```suggestion
  # for now, assume it's time if it's numeric and has a known time unit
```

Reply:
good catch, I updated the code, then forgot the comment